### PR TITLE
Mod: Cleantalk Status in list of sites

### PR DESCRIPTION
**Pull request by ArtemAnoshin**

### CODE REVIEW

File: inc/cleantalk-admin.php [hunk 1]
```diff
@@ -1216,3 +1216,49 @@ function apbct__add_debug_tab($links)
...
+/**
+ * Function for `wpmu_blogs_columns` filter-hook.
+ *
+ * @param string[] $sites_columns An array of displayed site columns.
+ *
+ * @return string[]
+ */
+function apbct__wpmu_blogs_columns_filter($sites_columns)
+{
+    $sites_columns[] = esc_html__('Cleantalk Status', 'cleantalk-spam-protect');
```
Comment (Glomberg) [inline inc/cleantalk-admin.php:1229]: The right spelling is CleanTalk

**Review by Glomberg — CHANGES_REQUESTED** [review]
Please, fix the codestyle.

File: inc/cleantalk-admin.php (hunk 1 above)
Comment (ArtemAnoshin) [reply inc/cleantalk-admin.php]: Done

**Review by Glomberg — APPROVED** [review]

File: inc/cleantalk-admin.php [hunk 2]
```diff
@@ -1216,3 +1216,49 @@ function apbct__add_debug_tab($links)
...
+ * @param string $_column_name The name of the column to display.
+ * @param int    $site_id     The current post ID.
+ *
+ * @return void
+ */
+function apbct__manage_sites_custom_column_action( $_column_name, $site_id )
+{
+    $cleantalk_data = get_blog_option($site_id, 'cleantalk_data');
+    $key_is_ok_text = esc_html__('The Access key is filled out and correct', 'cleantalk-spam-protect');
+    $key_is_bad_text = esc_html__('The Access key has not been filled or incorrect.', 'cleantalk-spam-protect');
```
Comment (alexandergull) [inline inc/cleantalk-admin.php:1248]: Please, use these phrases and no uppercase in the "access key". "The access key is set and correct."/"The access key is not set or is incorrect."

**Review by alexandergull — APPROVED** [review]

File: inc/cleantalk-admin.php (hunk 2 above)
Comment (ArtemAnoshin) [reply inc/cleantalk-admin.php]: DONE

**Pull request merged**